src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolution.cs

```diff
@@ -24,7 +23,7 @@ internal sealed class MutableConflictResolution
         private readonly RenamedSpansTracker _renamedSpansTracker;
 
         // List of All the Locations that were renamed and conflict-complexified
-        private readonly List<RelatedLocation> _relatedLocations;
+        public readonly List<RelatedLocation> RelatedLocations;
```

Comment:
should this be exposed in an immutable way? Even if they can't set the list instance, they can modify the contents.

Reply:
no.  this is actually mutate by callers.  MutableConflictResolution is intended to be mutable.  it changes as we're doing conflict resolution.  At hte end though we snap the final immutbale state and pass that back out.